include/appbase/application_base.hpp

```diff
@@ -3,15 +3,14 @@
 #include <appbase/abstract_plugin.hpp>
 #include <appbase/channel.hpp>
 #include <appbase/method.hpp>
-#include <boost/filesystem/path.hpp>
 #include <boost/core/demangle.hpp>
 #include <typeindex>
 #include <exception>
 #include <string_view>
+#include <filesystem>
 
 namespace appbase {
 namespace bpo = boost::program_options;
-namespace bfs = boost::filesystem;
```

Comment:
replacing this with `namespace fs = std::filesystem;` would make the code cleaner I think.

Reply:
I didn't do that just because this is part of the effort to replace all usages of boost and fc filesystem. Using `fs::` is just not visually distinctive to `fc::` enough for me.  I would prefer to keep `std::filesystem` unless you strongly object to it.

Reply:
I think we should keep all `using namespace` and namespace aliases out of header files.

Reply:
No strong objection.